boot/freeldr/freeldr/arch/uefi/uefimem.c

```diff
@@ -303,10 +314,23 @@ UefiExitBootServices(VOID)
     }
 }
 
+
+BOOLEAN
+MempSetupPaging(IN PFN_NUMBER StartPage,
+                IN PFN_NUMBER NumberOfPages,
+                IN BOOLEAN KernelMapping);
+
 VOID
 UefiPrepareForReactOS(VOID)
 {
+    ULONG* StackPtrNew;
+    ULONG* StackPtrEnd;
+    ULONG StackVar = 0;
+
+    /* Technically UEFI defines that stack space as 32kb */
+    StackPtrNew = &StackVar;
+    StackPtrEnd = (&StackVar + 0x2000);
```

Comment:
```suggestion
    StackPtrEnd = (StackPtrNew + 0x2000 / sizeof(StackPtrNew));
```
In order to fix error:
```
../src/boot/freeldr/freeldr/arch/uefi/uefimem.c:326:12: error: variable 'StackPtrNew' set but not used [-Werror=unused-but-set-variable]
     ULONG* StackPtrNew;
            ^~~~~~~~~~~
```